src/components/clusters/ClusterStatus.tsx

```diff
@@ -25,18 +26,28 @@ const getStatusIcon = (status: Cluster['status']) => {
 };
 
 type ClusterStatusProps = {
-  status: Cluster['status'];
-  statusInfo?: string;
+  cluster: Cluster;
 };
 
-const ClusterStatus: React.FC<ClusterStatusProps> = ({ status, statusInfo }) => {
+const ClusterStatus: React.FC<ClusterStatusProps> = ({ cluster }) => {
+  const { status, statusInfo, statusUpdatedAt } = cluster;
   const title = CLUSTER_STATUS_LABELS[status];
   const icon = getStatusIcon(status);
   const headerContent = React.useMemo(() => <div>{title}</div>, [title]);
```

Comment:
IMO all these useMemos are not needed and i'd say its actually _harmful_ to the app performance. These are simple values which do not need to be memoized.

Reply:
done